.changeset/fresh-cups-share.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap-cloud-sdk/connectivity': minor
+'@sap-cloud-sdk/http-client': minor
+---
+
+[Improvement] Default log output on the `info` level has been significantly reduced.
```

Comment:
[req] Please use the imperative in the changelog. You can find some examples in the changelog file in the root of the project.
@florian-richter will this be moved somewhere else with the new approach?

Reply:
Nothing changed on the changesets-side of things (other than requiring the tags to be correct later down the line)